routing_common/transit_serdes.hpp

```diff
@@ -219,5 +219,39 @@ class Deserializer
 private:
   Source & m_source;
 };
+
+template <typename Sink>
+class FixSizeNumberSerializer
```

Comment:
FixedSize

Reply:
Yes: `FixSizeSerializer` and `FixSizeDeserializer`

Reply:
s/FixSize/FixedSize :)

Reply:
Ops...

Reply:
`FixedSizeSerializer`/`FixedSizeDeserializer`